### DISCUSSION TITLE
webmanifest/favicon assets relocation to assets so as not to clobber/litter ?

### DISCUSSION THREAD

**Discussion by wolfspyre** [jpanther/congo#682] [General]
Does it make sense to relocate the congo default static favicon/webmanifest/etc files from static/ to assets/ and use `resources.Copy` to deploy them in `head.html:95-100` in the case that the user hasn't provided an override?
otherwise, the files in static get pushed to the webroot and subsequently overwritten, if, for example, someone was autogenerating site.webmanifest from a template.

it took me awhile to find this: https://dev.to/masakudamatsu/favicon-nightmare-how-to-maintain-sanity-3al7
but it's (IMO) a pretty good recent assessment of how-to-favicon

**Comment by jpanther** [comment]
I'm grouping this thread into a single issue to track this feature in one place - see https://github.com/jpanther/congo/issues/720.
